pkg/controller/controller_test.go

```diff
@@ -2478,6 +2490,7 @@ func assertServiceBindingStartingOrphanMitigation(t *testing.T, obj runtime.Obje
 	assertServiceBindingOperationStartTimeSet(t, obj, false)
 	assertServiceBindingReconciledGeneration(t, obj, originalBinding.Status.ReconciledGeneration)
 	assertServiceBindingOrphanMitigationSet(t, obj, true)
+	assertServiceBindingUnbindStatus(t, obj, originalBinding.Status.UnbindStatus)
```

Comment:
The `UnbindStatus` should be Required. If an orphan mitigation is in progress, then by definition an unbind is required.